### PR TITLE
Adding executable frame

### PR DESCRIPTION
**Pull request by tuhlmann**
This mainly keeps your code and adds the executable framework for Nest 6

### CODE REVIEW

**Review by BrunnerLivio — CHANGES_REQUESTED** [review]
Thanks a lot for your PR! It looks really got from what I have seen so far! :)

I think we should use TSLint and Prettier settings from [nestjs/typescript-starter](https://github.com/nestjs/typescript-starter/). Reason for that is because users will most likely copy and paste the code from this repo in their project. Most likely they will use the standard of NestJS, so it would be more convenient if it has the same settings and format options.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  "author": "Livio Brunner, Torsten Uhlmann",
+  "license": "UNLICENSED",
+  "scripts": {
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "lint": "tslint -p tsconfig.json -c tslint.json",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@lxdhub/db": "^1.10.1",
```
Comment (BrunnerLivio) [inline package.json]: We should remove `@lxdhub/db` and all the lines it refers to. That was the project I have used this technique for, but it is not related to this repository ;)

File: src/user/user.repository.ts [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
+    return await this.createQueryBuilder("user").getManyAndCount()
+  }
 }
 
-export const UserRepositoryProvider = {
-    provide: 'UserRepository',
-    useFactory: (connection: Connection) => connection.getCustomRepository(UserRepository),
-    inject: [Connection]
-};
+// export const UserRepositoryProvider = {
```
Comment (BrunnerLivio) [inline src/user/user.repository.ts]: Remove this.

This workaround was needed back in Nest 4 for custom repositories.

**Comment by tuhlmann** [comment]
@BrunnerLivio Awesome, will do the changes.

**Review by BrunnerLivio — APPROVED** [review]

**Comment by BrunnerLivio** [comment]
Thank so much @tuhlmann :)

**Pull request merged**